scout/build/case.py

```diff
@@ -115,52 +115,93 @@ def build_case(case_data, adapter):
     institute_obj = adapter.institute(institute_id)
     if not institute_obj:
         raise IntegrityError("Institute %s not found in database" % institute_id)
+
     case_obj["owner"] = case_data["owner"]
+    case_obj["smn_tsv"] = case_data.get("smn_tsv")
+    case_obj["collaborators"] = get_collaborators(case_data)
```

Comment:
The reason being that some times a `None` value is accepted in `case_obj` and sometimes not. I've tried my hardest not to change this when rewriting the code. That's why it will look different. I think we should accept None values. And I think that the discussion should be done elsewhere.

In this module I guess I'll rewrite the get-functions to set-functions for consistency.

Reply:
In this specific case collaborators can't be none, since at least one element is always present in the list (case owner).